service_mesh/v2x/ossm-custom-resources.adoc

```diff
@@ -13,6 +13,10 @@ You can configure your {ProductName} by modifying the default {ProductShortName}
 
 include::modules/ossm-cr-example.adoc[leveloffset=+1]
 
+include::modules/ossm-recommended-resources.adoc[leveloffset=+1]
```

Comment:
@Neal Timpe agree with your comments about the new content for recommended resources should go with a new assembly on performance and tuning.  I was thinking it should go with the High Availability content, once we finally write that.  But we'd discussed putting this content into this assembly until we write those other topics.  Are you working on performance and scalability with OSSMDOC-246?  Do you want me to just merge these modules and you can add them to your assembly?

Reply:
@JStickler I'm not sure. We may have discussed it, but now that I see the content, I don't think this is the right place for it. 

I'm not writing performance and scalability content in 246. The way I see it, it's more production configuration. I think it relates to this work you're doing with performance and scalability, but I think it would be okay to put it in its own assembly and add HA to it. 

It also might be a question for @heathjoy about where we should put this content while we build other content.

Reply:
I created a new assembly for the two new topics.  Let me know what you think.

Reply:
I think that makes a lot of sense. I think it needs to be added to topicmap.yml, but aside from that, it seems like the right thing.

Reply:
D'oh!  Added to topic map.